src/ParseServerRESTController.js

```diff
@@ -128,7 +128,14 @@ function ParseServerRESTController(applicationId, router) {
           })
           .then(
             resp => {
-              const { response, status } = resp;
+              let { response } = resp;
```

Comment:
It kind of smells that we have almost the same code in the Parse JS SDK. Is there a way around duplicating this?

Reply:
No

Reply:
Would it make sense to instead define a list of transformations from headers to response objects in the Parse JS SDK? This way both REST controllers could use it. Because after all this only seems relevant for the Parse JS SDK, so it seems logical to let it define this.

Or even better if we could move that logic one step after the REST controller, so that the SDK makes the transformation regardless of REST controller.

Reply:
Discussion continued in https://github.com/parse-community/parse-server/pull/8766#pullrequestreview-1647791411